src/main/java/nl/tudelft/scrumbledore/GUI.java

```diff
@@ -423,10 +441,22 @@ public void handle(KeyEvent keyPressed) {
 
         // Mapping the desired keys to the desired actions.
         if (keyPress.equals("LEFT")) {
+          // Logging the movement of the player to the game log.
+          Logger.log("Player moved to " + player.getPosition().getX() + ", "
+              + player.getPosition().getY());
```

Comment:
The keypress event handler doesn't tell much about the movement of the player. For example the Player could be standing next to a wall, and therefore the keypress would not result in any movement. Even if it did, it's coordinates would not be updated instantaniously, that takes a step. So wouldn't it be better to call logs like this for example "User pressed LEFT key" instead?